pkg/autodetect/main.go

```diff
@@ -216,3 +216,18 @@ func isElasticsearchOperatorAvailable(apiList *metav1.APIGroupList) bool {
 	}
 	return false
 }
+
+type matchingLabelKeys map[string]string
+
+func (m matchingLabelKeys) ApplyToList(opts *client.ListOptions) {
```

Comment:
Isn't cleaner to return `labelSelector` than modifying the struct?

Reply:
Would have been, but they changed the API and I couldn't find a cleaner way to do it. See https://github.com/operator-framework/operator-sdk/issues/2060